lr_face/experiments.py

```diff
@@ -75,7 +75,8 @@ def get_pairs_from_file(self, filename, cal_or_test):
         for pair_in_file in pairs_from_file:
             first = image_path_dict[pair_in_file[0]]
             second = image_path_dict[pair_in_file[1]]
-            pairs.append(FacePair(first, second))
+            if isinstance(first, FaceImage) and isinstance(second, FaceImage):
```

Comment:
is there no 'else'?

Reply:
when the image is not found (which means that the dataset that the image belongs to is not added in the params file) there is not really much we can do. But I'll add a print statement to attend to user if pairs are being skipped